docs/src/manual/multi.md

```diff
@@ -68,3 +68,56 @@ periods = TwoLevel(3, [5, 5, 10], [repr, repr, repr], 365.0);
 ```
 
 ## TwoLevelTree structure
+
+If there is uncertainty at a strategic level, this can be incorporated using the [`TwoLevelTree`](@ref) 
+time structure. This structure is represented by a tree, with each node corresponding to a strategic period that contains an operational time structure.
+
```

Comment:
```suggestion
The operational time structure can be any combination of the *[described structures](https://sintefore.github.io/TimeStruct.jl/stable/manual/basic/#Operational-time-structures)*

```
I do not like to have to provide a link like that. I would suggest that we can include proper labels for each heading that are consistent.
Examples could be:

- `man-oper`
  - `man-oper-simple`
  - `man-oper-calendar`
  - `man-oper-rep`
  - `man-oper-oper_scen`
- `man-multi`
  - `man-multi-twolevel`
  - `man-multi-twoleveltree`

Thoughts? I could do a full commit with these changes.

On a side note, I would also switch operational scenarios and representative periods in the documentation , so that people are aware of that operational scenarios can be within representative periods, but not the other way around.

Reply:
Proper labels would be nice - so just go ahead! I also agree with the change of order.

Reply:
Labels included in 0d8261c. I would suggest you take a look at them and greenlight them. Note that they are especially relevant for external referencing through `DocumenterInterLinks`. Hence we should **not** change them in a later stage.

Reply:
Labels looks fine to me.